roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -35,6 +35,25 @@ class DiskAvailability(OpenShiftCheck):
         },
     }
 
+    # recommended disk space for each location under an upgrade context
+    recommended_disk_upgrade_bytes = {
+        '/var': {
+            'masters': 10 * 10**9,
+            'nodes': 5 * 10 ** 9,
+            'etcd': 5 * 10 ** 9,
+        },
+        '/usr/local/bin': {
+            'masters': 1 * 10**9,
+            'nodes': 1 * 10**9,
+            'etcd': 1 * 10**9,
+        },
+        tempfile.gettempdir(): {
+            'masters': 0.5 * 10**9,
+            'nodes': 0.5 * 10**9,
+            'etcd': 0.5 * 10**9,
```

Comment:
Granted the 1 GB for /usr/local/bin and temp were both originally a rough guess, why do we make this different here?